src/puppet.ts

```diff
@@ -505,6 +506,26 @@ export abstract class Puppet extends PuppetEventEmitter {
   abstract tagContactList ()                                   : Promise<string[]>
   abstract tagContactRemove (tagId: string, contactId: string) : Promise<void>
 
+  /**
+   *
+   * Moment
+   *
+   */
+  abstract momentSignature (signature?: string)                              : Promise<boolean | string>
+  abstract momentCoverage (image: FileBox)                                   : Promise<boolean>
+  abstract postTextMoment (content: string, visibleList?: string[])          : Promise<string>
+  abstract postLinkMoment (urlLinkPayload: UrlLinkPayload, content?: string) : Promise<string>
+  abstract postImageMoment (images: FileBox[], content?: string)             : Promise<string>
+
+  abstract momentPayload (id: string)                                        : Promise<MomentPayload>
+
+  abstract momentList (option?: MomentListOption)                            : Promise<string[]>
+  abstract revokeMoment (id: string)                                         : Promise<boolean>
```

Comment:
Is `delete` or `remove` better than `revoke`?

Reply:
Please see https://cloud.google.com/apis/design/standard_methods and let's follow the google APIs recommendation for **standard methods**